obs-mcp/pkg/prometheus/guardrails.go

```diff
@@ -60,8 +71,9 @@ func ParseGuardrails(value string) (*Guardrails, error) {
 		case GuardrailDisallowBlanketRegex:
 			g.DisallowBlanketRegex = true
 		default:
-			return nil, fmt.Errorf("unknown guardrail: %q (valid options: %s, %s, %s)",
-				name, GuardrailDisallowExplicitNameLabel, GuardrailRequireLabelMatcher, GuardrailDisallowBlanketRegex)
+			return nil, fmt.Errorf("unknown guardrail: %q (valid options: %s, %s, %s, %s, %s)",
+				name, GuardrailDisallowExplicitNameLabel, GuardrailRequireLabelMatcher,
+				GuardrailDisallowBlanketRegex, GuardrailMaxMetricCardinality, GuardrailMaxLabelCardinality)
```

Comment:
IIUC `GuardrailMaxMetricCardinality` and `GuardrailMaxLabelCardinality` will still make the control flow fall under the `default` case? Since the two are accompanying the actual guardrails it may be worth keeping these out of the parsing logic?